templates/zerver/help/invite-new-users.md

```diff
@@ -34,8 +34,10 @@ invitation, but require them to authenticate via LDAP.
 
 1. Set **Are invitations required for joining the organization** to **No**.
 
+1. Find the section **Domain restrictions**.
+
```

Comment:
Not sure if it is right to say this as this is just below the "Joining the organization" section and will be clearly visible but I think we should make it clear that "Restrict email domains of new users?" is in different section. Is it fine or do we need to rephrase this or consider not adding this line?